cli/slsa-verifier/main_test.go

```diff
@@ -23,18 +30,10 @@ func pString(s string) *string {
 	return &s
 }
 
-// Versions of the builders to test.
-// TODO: Enable v1.0.0 for go builder
-var generatorVersions = map[string][]string{
-	"v0.0.2": {"go"},
-	"v1.1.1": {"go"},
-	"v1.2.0": {"generic"},
-}
-
 const TEST_DIR = "./testdata"
 
-func Test_runVerify(t *testing.T) {
-	t.Parallel()
+func Test_runVerifyGoAndGeneric(t *testing.T) {
```

Comment:
Shall this be more generic and be called `Test_runVerifyArtifactPath`, and the other `Test_runVerifyArtifactReference`?